packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -1053,7 +1049,7 @@ export class DeltaManager
         assert(!readonly || this.connectionMode === "read", "readonly perf with write connection");
         this.set_readonlyPermissions(readonly);
 
-        this.cancelDelayInfo(this.deltaStreamDelayId);
+        this.cancelDelayInfo();
```

Comment:
Maybe it should not be called cancelled, but "refresh"?
Also the intention of this call to send an "off" message to throttling warning, so we should try to name it accordingly.

Reply:
Renamed.